R/MixedModelsMessages.R

```diff
@@ -0,0 +1,209 @@
+    ),
+    iterations
+  )
+}
+.mmMessageLowBMFI       <- function(nChains) {
+  sprintf(
+    ngettext(
+      nChains,
+      "Bayesian Fraction of Missing Information (BFMI) that was too low in %i chain indicating that the posterior distribution was not explored efficiently. Try increasing number of 'Burnin' and 'Iterations'.",
+      "Bayesian Fraction of Missing Information (BFMI) that was too low in %i chains indicating that the posterior distribution was not explored efficiently. Try increasing number of 'Burnin' and 'Iterations'."
+    ),
+    nChains
+  )
+}
+.mmMessageMaxTreedepth  <- function(iterations) {
+  sprintf(
+    ngettext(
+      iterations,
+      "The Hamiltonian Monte Carlo procedure might be inefficient -- %i transition exceeded the maximum tree depth. This can be solved by carefully increasing 'Maximum tree depth",
+      "The Hamiltonian Monte Carlo procedure might be inefficient -- %i transitions exceeded the maximum tree depth. This can be solved by carefully increasing 'Maximum tree depth"
+    ),
+    iterations
+  )
+}
+.mmMessageMaxRhat       <- function(Rhat) {
+  gettextf(
+    "Inference possibly unreliable -- MCMC chains might not have converged; The largest R-hat is %.3f > 1.01. To lower R-hat please increase 'Iterations', or 'Adapt delta' in the Options section.",
+    Rhat
+  )
+}
+.mmMessageMinESS        <- function(ESS, treshold) {
+  gettextf(
+    "Low estimation accuracy -- The smallest Effective Sample Size (ESS) is %.2f < %1.0f. To increase accuracy please increase 'Iterations', or 'Adapt delta' in the Options section.",
+    ESS,
+    treshold
+  )
+}
+.mmMessageBadWAIC       <- function(n_bad) {
+  sprintf(
+    ngettext(
+      n_bad,
+      "WAIC estimate unreliable -- There was %1.0f p_waic estimate larger than 0.4. We recommend using LOO instead.",
+      "WAIC estimate unreliable -- There were %1.0f p_waic estimates larger than 0.4. We recommend using LOO instead."
+    ),
+    n_bad
+  )
+}
+.mmMessageBadLOO        <- function(n_bad) {
+  sprintf(
+    ngettext(
+      n_bad,
+      "LOO estimate unreliable -- There was %1.0f observation with the shape parameter (k) of the generalized Pareto distribution higher than > .5.",
+      "LOO estimate unreliable -- There were %1.0f observations with the shape parameter (k) of the generalized Pareto distribution higher than > .5."
+    ),
+    n_bad
+  )
+}
+.mmMessageFitType       <- function(REML) {
+  gettextf("The model was fitted using %1$s.%2$s",
+           ifelse(REML, gettext("restricted maximum likelihood"), gettext("maximum likelihood")),
+           ifelse(REML, gettext(" Please note that models with different fixed effects cannot be compared when REML is used. To use ML, switch 'Test model terms' to 'Likelihood ratio tests'."), ""))
+}
```

Comment:
if you can avoid `gettextf` it makes the work of the translators a lot easier.
```r
if (REML) {
  gettext("The model was fitted using restricted maximum likelihood. Please note that models with different fixed effects cannot be compared when REML is used. To use ML, switch 'Test model terms' to 'Likelihood ratio tests'.")
} else {
  gettext("The model was fitted using maximum likelihood.")
}
```

Reply:
ok, changed